Add caution against updating (#941)
pi-hole/docs: docs/ftldns/compile.md

```bash
sudo service pihole-FTL restart
```
## Caution

> Once your customized *FTL*DNS binary is installed, do not run "pihole -up" or "pihole checkout." These commands may call update.sh and basic-install.sh, which will trigger an overwrite of your customized *FTL*DNS binary with Pi-hole's pre-compiled *FTL*DNS binary.

Reviewer: ```suggestion
Once your customized *FTL*DNS binary is installed, do not run `pihole -up` or `pihole checkout`. These commands can trigger an overwrite of your customized *FTL*DNS binary with Pi-hole's pre-compiled *FTL*DNS binary.
```

sudo service pihole-FTL restart
```

## Caution

> Once your customized *FTL*DNS binary is installed, do not run `pihole -up` or `pihole checkout`. These commands can trigger an overwrite of your customized *FTL*DNS binary with Pi-hole's pre-compiled *FTL*DNS binary.

Reviewer 2: ```suggestion
Once your homebrew `pihole-FTL` binary is built and installed, do not run `pihole -up` or `pihole checkout`. These commands might overwrite of your local `pihole-FTL` binary with Pi-hole's pre-compiled binaries.
```
